packages/clarity-js/types/data.d.ts

```diff
@@ -97,6 +97,12 @@ export const enum Dimension {
     PageLanguage = 16
 }
 
+export const enum Region {
```

Comment:
>Region [](http://example.com/codeflow?start=18&length=6)

I thought we will not send down the region list in code - and this would be a mapping maintained by the server

Reply:
Sounds good. Moved it to dev tools to serve as an example and removed it from here for now.